### PR TITLE
Adding Vonage connector

### PR DESCRIPTION
**Pull request by matt-lethargic**
Adding Certified Vonage connector

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Vonage/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Vonage Connector
```
Comment (v-eisljami) [inline certified-connectors/Vonage/README.md:1]: Readme doenst have all required sections. Please refer to this [link ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for a readme template.

File: certified-connectors/Vonage/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+              6
+            ],
+            "in": "formData",
+            "name": "code_length",
+            "type": "integer"
+          },
+          {
+            "default": "en-us",
+            "description": "By default, the SMS or text-to-speech (TTS) message is generated in the locale that matches the `number`. For example, the text message or TTS message for a `33*` number is sent in French. Use this parameter to explicitly control the language used for the Verify request. A list of languages is available: <https://developer.vonage.com/verify/guides/verify-languages>",
+            "x-ms-summary": "By default, the SMS or text-to-speech (TTS) message is generated in the locale that matches the `number`. For example, the text message or TTS message for a `33*` number is sent in French. Use this parameter to explicitly control the language used for the Verify request. A list of languages is available: <https://developer.vonage.com/verify/guides/verify-languages>",
```
Comment (v-eisljami) [inline certified-connectors/Vonage/apiDefinition.swagger.json:330]: Please check this summary. These summaries should not exceed 80 characters limit. That's why the swagger validation fails.

**Review by v-eisljami — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@matt-lethargic 
I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

File: certified-connectors/Vonage/README.md (hunk 1 above)
Comment (matt-lethargic) [reply certified-connectors/Vonage/README.md:1]: Looking at these now

File: certified-connectors/Vonage/apiDefinition.swagger.json (hunk 2 above)
Comment (matt-lethargic) [reply certified-connectors/Vonage/apiDefinition.swagger.json]: Addressed all summaries that were over 80 chars

File: certified-connectors/Vonage/README.md (hunk 1 above)
Comment (matt-lethargic) [reply certified-connectors/Vonage/README.md:1]: I have updated this with all required sections

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**